src/index.json

```diff
@@ -17635,6 +17635,49 @@
                     "version": "0.2.6"
                 },
                 "sha256Digest": "0a0e897028ecce7ab26b55a4971177d8a7df23f15c772bae2000447380b3d1e2"
+            },
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/SelfServiceVM/CLI/azurestackhci-0.2.9-py3-none-any.whl",
+                "filename": "azurestackhci-0.2.9-py3-none-any.whl",
```

Comment:
What's the difference with the `stack-hci` extension https://github.com/Azure/azure-cli-extensions/tree/main/src/stack-hci

Reply:
We are working on porting azurestackhci to stack-hci. Once is it ported, we will deprecate azurestackhci. So, basically there is no difference, we just renamed  azurestackhci to stack-hci.